File: cpp/funkcje.cpp
/*
 * funkcje.cpp
 */


#include <iostream>

using namespace std;

void dodaj (int a, int b) //  void nie zwraca wyniku (deklaracja funkcji)
{
    cout << "Suma: " << a + b << endl;
}

void iloczyn (int a , int b)
{
    cout << "Iloczyn: " << a * b << endl; 
}


int odejmij (int l1, int l2)
{
    return l1 - l2;  // zwraca wynik - return
}

int iloraz (int a, int b)
{
    if (b==0){
    cout << "Nie dziel przez 0!" << endl;
    return 0;
    }
    return a / b; 
}

int main(int argc, char **argv)
{
    int a, b;
    a = b = 0;
    cout << "Podaj 2 liczby: " << endl;
    cin >> a >> b;
    
    dodaj (a, b); // wywołanie funkcji
    cout << "Różnica: " << odejmij(a, b) << endl;
    
    iloczyn (a, b);
    
   cout << "Iloraz: " << iloraz(a, b) << endl;
    
    
    
    return 0;
}

File: Python/funkcje.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-


def sumuj(x, y):
    return x + y


def odejmij(x, y):
    return x - y


def iloczyn(x, y):
    return x * y


def iloraz(x, y):
    return x / y


def main(args):
    a = int(input('Podaj liczbę: '))
    print(a)
    b = int(input('Podaj liczbę: '))
    print(b)

    print('Suma:', sumuj(a, b))
    print ('Różnica:', odejmij(a, b))
    print ('Iloczyn:', iloczyn(a, b))
    print ('Iloraz', iloraz(a, b))

    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
